docs/source/architecture/poet.rst

```diff
@@ -65,7 +65,7 @@ provided by other lottery algorithms. The probability of election
 is proportional to the resources contributed (in this case, resources
 are general purpose processors with a trusted execution environment).
 An attestation of execution provides information for verifying that the
-certificate was created within the enclave (and that the validator waited
+Timer was created within the enclave (and that the validator waited
```

Comment:
`certificate` could be the right word here. Each validator has to confirm to other validators that they waited for random amount of time that they're claiming in the certificate.

@dcmiddle to confirm